README.md

```diff
@@ -20,7 +20,7 @@ An animation propert was also added in the nav bars in the header section to ani
 
 ### Install
 
-- Copy the github link for the repository [repository link](https://github.com/Ngufor-emmanuella/personal-portfolio.git)
+- Copy the github link for the repository [repository link]( https://ngufor-emmanuella.github.io/personal-portfolio/)
```

Comment:
### OPTIONAL

- [ ] Kindly move the live demo link to the deployment section so users can easily find the live demo link 👍🏿 


- You can change your repository link to link of your project repository 👍🏿

Reply:
thanks already did the changes